README.md

```diff
@@ -0,0 +1,27 @@
+# Half-Life SDK for Xash3D
+
+Half-Life SDK for Xash3D with some fixes.
+
+## How to build
+
+### Windows
+
+TODO
+
+### Linux
+
+TODO
+
+### OS X
+
+TODO
+
+### FreeBSD
+
+```
+    cd dlls
+    gmake CXX=clang++ CC=clang
```

Comment:
I'm on FreeBSD, and even with ?= I need to explicitly set CXX to clang++, because it still considers g++ (which does not exist, since executable name now is g++48)